src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -58,11 +58,16 @@
 	$classes[] = 'tribe-events-calendar-month__multiday-event--hidden';
 }
 
+$event_start_datetime = strtotime( tribe_get_start_date( $event->ID, true, Tribe__Date_Utils::DBDATETIMEFORMAT ) );
```

Comment:
Use `$event->dates->start->format( Dates::DBDATETIMEFORMAT )` here.
Add `use Tribe__Date_Utils as Dates` at the top of the file to clean up the code a bit.

Reply:
👍